_shared/project/requirements/tests.in

```diff
@@ -1,8 +1,14 @@
 pip-tools
 pip-sync-faster
 -r prod.txt
-coverage[toml]
 pytest
+pytest-coverage
+{% if cookiecutter.get("__parallel_unit_tests") %}
+pytest-xdist[psutil]
+{% if cookiecutter.get("postgres") == "yes" %}
+filelock
```

Comment:
`filelock` is only needed if the project runs its tests in parallel and has a DB